### DISCUSSION TITLE
New suggested answers is confused

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#50142] [Discussions] [answered]
### Select Topic Area

Bug

### Body

- https://github.com/community/community/discussions/49954

![image](https://user-images.githubusercontent.com/2119212/225363913-1e30a0a7-b58d-4b4d-a981-f22f240ccde2.png)

Problems:
1. There's an arrow pointing down, but I've reached the end / past the end of the discussion.
2. I'm pretty sure the thing that brought me there was newer than the concept's 8...

**Comment by jsoref** [comment]
- https://github.com/community/community/discussions/49954#discussioncomment-5335107

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/2119212/225646528-c1c6316a-01cf-4968-aac6-51bc25c21059.png">

**Comment by LindseyB** [comment, marked as answer]
Thanks for the report. This is definitely a bug.

**Reply by jsoref** [reply]
@LindseyB were you answering on behalf of github? I assumed you were.

**Reply by LindseyB** [reply]
I was, but I no longer work there as of 2025

**Reply by jsoref** [reply]
Thanks. Sorry, there's been quite a lot of spam in this community from people speaking as if they are github staff when they aren't.
